src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/AlleleSubsettingUtils.java

```diff
@@ -94,7 +94,7 @@ public static GenotypesContext subsetAlleles(final GenotypesContext originalGs,
                 newLog10GQ = -0.1*g.getGQ();
             }
 
-            final boolean useNewLikelihoods = !suppressUninformativePLs ||
+            final boolean useNewLikelihoods = forceEmitUninformativePLs ||
```

Comment:
The old conditional wasn't right -- I trusted the IntelliJ refactoring too much.  This is also a more logical name.